Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -53,9 +54,9 @@ public void setupClimbingCommands(
             AscendCommand ascend,
             RopeAlignerCommand aligner)   
     {
-        oi.leftButtons.getifAvailable(2).whileHeld(descend);
-        oi.rightButtons.getifAvailable(2).whileHeld(ascend);
-        oi.rightButtons.getifAvailable(5).whileHeld(aligner);
+        descend.includeOnSmartDashboard("Climber descend");
+        ascend.includeOnSmartDashboard("Climber ascend");
```

Comment:
I think these commands need to be whileHeld - just because if we don't stop them in time, we could damage the machine.